site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -4,6 +4,21 @@ ifeval::["{base_field_name}" != "tenant"]
 The list of Tenant objects.
 endif::[]
 
+[field]#{base_field_name}.captchaConfiguration.captchaMethod# [type]#[String]# [since]#Available since 1.30.0#::
+The type of captcha method to use.
+
+[field]#{base_field_name}.captchaConfiguration.enabled# [type]#[Boolean]# [since]#Available since 1.30.0#::
+Whether captcha configuration is enabled.
+
+[field]#{base_field_name}.captchaConfiguration.secretKey# [type]#[String]# [since]#Available since 1.30.0#::
+The secret key for this captcha method.
+
+[field]#{base_field_name}.captchaConfiguration.siteKey# [type]#[String]# [since]#Available since 1.30.0#::
+The site key for this captcha method.
+
+[field]#{base_field_name}.captchaConfiguration.threshold# [type]#[Float]# [since]#Available since 1.30.0#::
+The threshold which separates passing from failing the Google v3 or HCaptcha Enterprise score. If not using either of those methods then this value is ignored.
```

Comment:
```suggestion
The numeric threshold which separates a passing score from a failing one. This value only applies if using either the Google v3 or HCaptcha Enterprise method, otherwise this value is ignored.
```